crates/miden-testing/src/kernel_tests/tx/test_epilogue.rs

```diff
@@ -418,3 +428,15 @@ fn test_epilogue_increment_nonce_violation() {
     );
     assert_execution_error!(process, ERR_ACCOUNT_NONCE_DID_NOT_INCREASE_AFTER_STATE_CHANGE)
 }
+
+#[test]
+fn test_epilogue_execute_empty_transaction() {
+    let tx_context = TransactionContextBuilder::with_standard_account(ONE).build();
```

Comment:
Should we also add a test that checks that a transaction that produces one output note (without assets) using the tx script is also considered an empty transaction and fails with the same error?